test/smoke_test.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -x
+set -e
+set -a
+
+NODE_MODE=$1 #single or split mode
+
+COMPOSE_CMD="NODE_IMAGE=$IMAGE docker-compose -p $NODE_MODE -f test/docker-compose.$NODE_MODE.yaml"
+
+# Up compose
+/bin/bash -c "$COMPOSE_CMD up -d --wait --quiet-pull"
+
+# set tests variables and run tests
+GITHUB_VARS=$(env | awk -F '=' -v ORS==" " '/^GITHUB_/ {print "-e " $1 "=" $2}')
+RAND_NUM=$(echo $RANDOM$RANDOM$RANDOM | cut -c 1-10)
+
+RUN_TESTS="pytest pytest"
+if [ "$ALLURE_UPLOAD_REPORT" = "true" ]; then
+  RUN_TESTS="pytest allurectl watch --job-uid $RAND_NUM -- pytest"
+fi
+
+NODE_UUID=$(/bin/bash -c "$COMPOSE_CMD exec node cat /opt/wallarm/etc/wallarm/node.yaml | grep uuid | awk '{print \$2}'")
+PYTEST_CMD="$COMPOSE_CMD exec $GITHUB_VARS -e NODE_UUID=$NODE_UUID $RUN_TESTS -n $PYTEST_WORKERS $PYTEST_ARGS"
+
+set +e #we need to get logs anyway, even if the tests were not successful.
+/bin/bash -c "$PYTEST_CMD"
+TESTS_STATUS=$?
+
+set -e
+
+# Get logs from containers
+if [ ! -d ./test/logs/$NODE_MODE ]; then
+  mkdir -p ./test/logs/$NODE_MODE/node
+fi
+
+/bin/bash -c "$COMPOSE_CMD cp node:/opt/wallarm/var/log/wallarm ./test/logs/$NODE_MODE/node"
+/bin/bash -c "$COMPOSE_CMD logs --no-color node > ./test/logs/$NODE_MODE/node/node.log"
+
+if [[ $NODE_MODE == "split" ]]; then
+  mkdir -p ./test/logs/$NODE_MODE/post-analytics
+  /bin/bash -c "$COMPOSE_CMD cp post-analytics:/opt/wallarm/var/log/wallarm ./test/logs/$NODE_MODE/post-analytics"
+  /bin/bash -c "$COMPOSE_CMD logs --no-color post-analytics > ./test/logs/$NODE_MODE/post-analytics/post-analytics.log"
+fi
+
+if [ ! -z ${CI+x} ];
+then
+    echo "We need to archive logs directory if tests are running in CI"
+    tar -czvf node-logs-$TEST_ARCH-$NODE_MODE.tar.gz test/logs
+fi
+
+# down compose
+/bin/bash -c "$COMPOSE_CMD down"
```

Comment:
why do we use `/bin/bash -c` everywhere instead of directly calling the command?